src/components/Toolbar/Toolbar.tsx

```diff
@@ -1,27 +1,68 @@
 import React from 'react';
 import { useTranslation } from '../../hooks/useTranslation/useTranslation';
+import { AnswerModeType } from '../../types/types';
 
 type ToolbarProps = {
   title: string;
-  enableSettings: boolean;
-  showSettings: boolean;
-  toggleShowSettings: () => void;
+  activeAnswerMode: AnswerModeType;
+  enableAnswerMode: boolean;
+  enableLanguageMode: boolean;
+  handleAnswerModeChange: () => void;
+  handleLanguageModeChange: () => void;
```

Comment:
We haven't talked much about this and I definitely haven't followed my own rules here, but there's a convention to name event function props `onX`. The function itself is often called `handleX`.

```tsx
const Parent = () => {
  const handleUpdate = (data: Data) => { ... };

  return <Child onUpdate={handleUpdate} />;
};
```

We don't have to fix this now (or ever), but we might want to follow this convention in the future.

Reply:
Added